src/main/java/net/coderbot/iris/config/IrisConfig.java

```diff
@@ -121,7 +131,8 @@ public void load() throws IOException {
 		properties.load(Files.newInputStream(propertiesPath));
 		shaderPackName = properties.getProperty("shaderPack");
 		enableShaders = !"false".equals(properties.getProperty("enableShaders"));
-		enableDebug = !"false".equals(properties.getProperty("enableDebug"));
+		enableDebug = "true".equals(properties.getProperty("enableDebug"));
```

Comment:
You're probably just going to need to rename the option at this point since so many people now have enableDebug=true in their config files from this oversight.